test/fit_tests/README.md

```diff
@@ -0,0 +1,123 @@
+# FIT Test Overview
+
+The FIT test suite is an open-source testing harness for RackHD and OnRack software.
+RackHD/OnRack (https://github.com/RackHD) is the open-sourced Hardware Management and Orchestration
+software developed by EMC for datacenter administration.
+
+FIT stands for Functional Integration Tests and is intended for Continuous Integration testing
+as well as standalone testing. It was originally developed by the EMC OnRack test group to provide
+the RackHD community a flexible test and deployment toolset that will work in a variety of
+valid configurations and environments.
+
+FIT is written in Python 2.7 'unittest' and uses Nose (nosetests) as the test runner.
+
+# Running FIT
+
+## Requirements and Setup
+
+FIT tests are intended to be run on Ubuntu 14+ Linux.
+Test harness may be run on appliance host (localhost), or third party machine.
+Deployment scripts must be run under third party Ubuntu Linux host.
+Tests require the following virtual environment commands be executed:
+
+    virtualenv .venv
+    source .venv/bin/activate
+    pip install -r requirements.txt
+
+
+## Organization
+
+- 'tests' is the test 'harness'
+- 'common' contains any common library functions
+- 'deploy' contains deployment and installation scripts
+- 'templates' contains script templates for making new tests
+- 'util' contains non-test utilities
+
+## Configuration
+
+Local runtime parameters are set from the 'config/global_config.json' file.
+Stack definitions are set from the 'config/stack_config.json' file.
+An alternate configuration directory can be selected using the -config argument.
+More details in config/README.mg.
+
+## Running the tests
+
+All FIT tests can be run from the wrapper 'run_tests.py':
+
+    usage: run_tests.py [-h] [-test TEST] [-config CONFIG] [-group GROUP]
+                        [-stack STACK] [-ora ORA] [-version VERSION] [-xunit]
+                        [-list] [-sku SKU] [-obmmac OBMMAC | -nodeid NODEID]
+                        [-http | -https] [-v V]
+
+    Command Help
+
+    optional arguments:
+      -h, --help        show this help message and exit
+      -test TEST        test to execute, default: tests/
+      -config CONFIG    config file location, default: fit_tests/config
+      -group GROUP      test group to execute: 'smoke', 'regression', 'extended',
```

Comment:
Groups are just tags that are set in the test script with an '@attr' decorator.

Example:
@attr(all=True, regression=True)
class os_bootstrap_base(fit_common.unittest.TestCase):

Nosetests does a search on the attr tag and runs any that match.